src/components/Form.js

```diff
@@ -39,11 +39,12 @@ const Form = () => {
   };
   return (
     <>
-      <form onSubmit={handleAddBook}>
-        <h3>ADD NEW BOOK</h3>
+      <form className="form-container" onSubmit={handleAddBook}>
+        <h3 className="form-title">ADD NEW BOOK</h3>
```

Comment:
- [ ] I think it would be great if you add a category beside the  author input since it is better to have it to let the user choose a category  from the categories displayed so I highly recommend to add it beside the author input since it is on the [Zeplin](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) design and it is very important to make your final design  should match Zeplin's design (this includes all elements + the same styling) as required 💪🏻🤩✅🔥💯

## it says all the element should be there and the input is one of these elements 😊🚀
![image](https://user-images.githubusercontent.com/58553711/230515199-e00abdd6-4025-4a5b-b1c9-5d823f110cdc.png)

Reply:
Changes applied